fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
I don't care much about the error message, imo we should just keep it simple. Just offered a suggestion in response to @bnoordhuis's concerns. I don't think anyone is testing for `err.code === 'MODULE_NOT_FOUND'`, so for the rare person who does manage to encounter this error, I think just telling them plainly how they've gone wrong is the best bet.

> Can you stipulate your conditions for that?

From my perspective, resolve the 0.10 concerns and demonstrate the change will work as intended on all the major platforms. I think/hope we're pretty close to figuring that out. I would want to publish this as a patch release, but we need to be extra careful not to cause a problem for users by doing that. A LOT of people will appreciate this issue quietly and seamlessly disappearing.